sql/index.ts

```diff
@@ -142,6 +142,19 @@ export class Sql {
     return `unix_millis(${timestamp})`;
   }
 
+  public timestampCurrentUTC() {
```

Comment:
```suggestion
  public currentUtcTimestamp() {
```

Reply:
Have moved all timestamp functions into a `timestamps` property and new class which makes them nicer to read.